starter-code/src/Contacts.jsx

```diff
@@ -0,0 +1,74 @@
+import React, {Component} from 'react';
+import {getFirstFive} from './contactsList';
+import {getContacts} from './contactsList';
+
+class Contacts extends Component {
+  state = {
+    contacts: getFirstFive (),
+    fulllist: getContacts()
+  };
+
+  handleDelete = contact => {
+    //let contacts = [...this.state.contacts];
```

Comment:
Overall your methods are very clean and well written, well done! 

Small remark on this line is to remember to remove all out-commented code before you hand in the assignment.